.gitignore

```diff
@@ -360,4 +360,10 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
+/AzureSentiment.sln
+/WikiDBUpdaterHttp/host.json
+/WikiDBUpdaterHttp/WikiDBUpdaterHttp.csproj
+/WikiDBUpdaterTimer/WikiDBUpdaterTimer.csproj
+/WikiSentiment/WikiSentiment.csproj
```

Comment:
csproj and .sln are absolutely not .gitignore files, but an integral part of the project